### PR TITLE
update NS to 2019 boundaries

### PR DESCRIPTION
**Pull request by DrMeers**

### CODE REVIEW

File: boundaries/ocd-division/country:ca/province:ns/2019/definition.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import re
+from datetime import date
+
+import boundaries
+
+boundaries.register('Nova Scotia electoral districts',  # (2019)
```
Comment (jpmckinney) [inline boundaries/ocd-division/country:ca/province:ns/2019/definition.py:6]: Also, edit the old boundaries to put (2012) in the name. See https://github.com/opennorth/represent-canada/issues/102

```suggestion
boundaries.register('Nova Scotia electoral districts (2019)',
```

Comment (DrMeers) [reply boundaries/ocd-division/country:ca/province:ns/2019/definition.py]: Thanks @jpmckinney -- how's that now?

**Comment by jpmckinney** [comment]
I've deployed the new boundaries and reconfigured the representative set to use it. Thanks!

**Comment by DrMeers** [comment]
> I've deployed the new boundaries and reconfigured the representative set to use it. Thanks!

Thank you @jpmckinney !

When you say "reconfigured the representative set to use it", was that something I should have done and/or should do for next time? Where is the code/config in question?

**Comment by jpmckinney** [comment]
In the Django admin, the representative set needs to be updated to point to the correct boundary set.

**Comment by DrMeers** [comment]
Ah OK, so that's a database-level thing that I don't have access to via the codebase

**Pull request merged**